src/routes/comment.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+
+const router = express.Router();
+
+const commentController = require('../controllers/comment');
+const postController = require('../controllers/comment');
+
+router.post('/comment/:id', postController.addComment);
```

Comment:
`/api/comment` will already be added in front, so another `comment` is not required in the route.